examples/common/python/utility/signature.py

```diff
@@ -256,7 +263,10 @@ def generate_client_signature(self, input_json_str,
 
         #Update the input json params
         input_json_params["encryptedRequestHash"] = encrypted_request_hash_str
-        input_json_params['requesterSignature'] = self.generate_signature(final_hash, private_key)
+        signature = self.generate_signature(final_hash, private_key)
+        if signature == SignatureStatus.FAILED:
+            return SignatureStatus.FAILED
```

Comment:
Better to return consistent types on success and failure. One option is to return a tuple with (signature, status). Also please update function comments accordingly

Reply:
Done